packages/rs-drive/src/drive/document/update.rs

```diff
@@ -636,6 +642,39 @@ impl Drive {
         }
         Ok(batch_operations)
     }
+
+    /// Add update multiple documents operations
+    pub fn add_update_multiple_documents_operations<'a>(
+        &self,
+        documents: &'a [DocumentStub],
+        data_contract: &'a DataContract,
+        document_type: &'a DocumentType,
+        drive_operations: &mut Vec<DriveOperationType<'a>>,
```

Comment:
This is confusing, you want to add to drive_operations, however we use drive_operations to mean something other than drive_operation types in most places. I would rename to drive_operations_types. And we will find better naming one day around this problem.